markdown/3.0/en/ff-searchbox.md

```diff
@@ -24,58 +24,73 @@ The following code-example shows the aforementioned configuration.
 ```
 For more information, see the [API reference](/api/ff-searchbox#tab=api).
 
-## Extending the input
-The `ff-searchbox` extends a normal `<input>`. Simply add the attribute
-`is="ff-searchbox"` to a standard `<input>` element. On ENTER a search
-query is fired with the current value.
+## Wrapping input element with ff-searchbox
+The `ff-searchbox` wraps a regular HTML `<input>` element. Simply place
+`<input>` between `<ff-searchbox>` tags. On ENTER a search query is fired
```

Comment:
maybe add "...a search query is fired _to FACT-Finder_"

Reply:
"... a search query to FACT-Finder is fired..." - please check now.